### PR TITLE
fix zero division error, add check and also fix multipolygon issue #11

### PR DESCRIPTION
**Pull request by Hagellach37**
I looked at the groups creation and added few lines to avoid zero division errors.

### CODE REVIEW

File: import_module/create_groups.py [hunk 1]
```diff
@@ -205,11 +209,15 @@ def get_horizontal_slice(extent, geomcol, zoom):
...
+                slice_infos['tile_y_bottom'].append(TileY+3)
+                slice_infos['slice_collection'].AddGeometry(sliced_poly)
                 #logging.info('sliced poly: %s' % sliced_poly)
             elif sliced_poly.GetGeometryName() == 'MULTIPOLYGON':
                 for geom_part in sliced_poly:
-                    slice_collection.AddGeometry(geom_part)
+                    slice_infos['tile_y_top'].append(TileY)
+                    slice_infos['tile_y_bottom'].append(TileY + 3)
+                    slice_infos['slice_collection'].AddGeometry(geom_part)
                     #logging.info('sliced poly: %s' % sliced_poly)
```
Comment (ericboucher) [inline import_module/create_groups.py]: Would these be helpful to debug? If so, maybe switch to logging.debug and uncomment?

File: import_module/create_groups.py [hunk 2]
```diff
@@ -261,38 +271,32 @@ def get_vertical_slice(geomcol, zoom):
...
-            TileY_bottom += 3
-
         # get lower right tile coordinates
         pixel = lat_long_zoom_to_pixel_coords(ymin, xmax, zoom)
         tile = pixel_coords_to_tile_address(pixel.x, pixel.y)
         TileX_right = tile.x
 
+
+        # we don't compute tile y top and tile y botton coordinates again, but get the ones from the list
+        # doing so we can avoid problems due to rounding errors and resulting wrong tile coordinates
```
Comment (ericboucher) [inline import_module/create_groups.py]: and resulting IN wrong tile coordinates

File: import_module/create_groups.py (hunk 1 above)
Comment (Hagellach37) [reply import_module/create_groups.py]: good question. I think, I never used this info in the last times. so maybe I will just remove it.

File: import_module/create_groups.py (hunk 2 above)
Comment (Hagellach37) [reply import_module/create_groups.py]: +1

**Pull request merged**